engine/apps/user_management/tests/test_user.py

```diff
@@ -13,7 +10,7 @@ def test_self_or_admin(
     organization = make_organization()
     admin = make_user_for_organization(organization)
     second_admin = make_user_for_organization(organization)
-    editor = make_user_for_organization(organization, role=Role.EDITOR)
+    editor = make_user_for_organization(organization)
```

Comment:
this currently grants all permissions, should probably update this by specifying a `permissions` kwarg